Doc/whatsnew/3.14.rst

```diff
@@ -617,6 +617,15 @@ sys.monitoring
 Two new events are added: :monitoring-event:`BRANCH_LEFT` and
 :monitoring-event:`BRANCH_RIGHT`. The ``BRANCH`` event is deprecated.
 
+
+time
+----
+
+* Ensure that :func:`time.sleep(0) <time.sleep>` does not degrade over time
```

Comment:
I suppose that you're referring to performance:

```suggestion
* Ensure that :func:`time.sleep(0) <time.sleep>` performance does not degrade over time
```

Reply:
Depending on the resolution of this PR (namely, whether we eventually use `select()`), I'll amend the NEWS so I won't commit that suggestion for now.